metaci/release/utils.py

```diff
@@ -13,12 +13,17 @@
 logger = logging.getLogger(__name__)
 
 
-def implementation_payload(role, config_item):
-    if role and config_item:
+def implementation_payload(role, config_item, release):
+    if role and config_item and release:
         return {
             "description": role,
-            "start_time": role,
-            "end_time": role,
+            "owner": "00XXXXXXXXXX",  # need to tie GUS user id to MetaCI user.
+            "start_time": release.implementation_steps.get(
+                plan__role=f"{role}"
```

Comment:
There's no reason to use an f-string if it's already a string

```suggestion
                plan__role=role
```